src/moonlight-server/state/data-structures.hpp

```diff
@@ -38,6 +38,23 @@ enum STANDARD_PORTS_MAPPING {
   RTSP_SETUP_PORT = 48010
 };
 
+inline int get_port(STANDARD_PORTS_MAPPING port) {
+  switch (port) {
+  case HTTPS_PORT:
+    return utils::get_env("WOLF_HTTP_PORT") ? std::stoi(utils::get_env("WOLF_HTTP_PORT")) : HTTPS_PORT;
```

Comment:
Should use `WOLF_HTTPS_PORT`.

Reply:
My bad, sorry for the typo. I've just pushed a fix for it, thanks for testing!